Formula/openshift-cli.rb

```diff
@@ -51,13 +49,12 @@ def install
   end
 
   test do
-    version_output = shell_output("#{bin}/oc version --client 2>&1")
-    assert_match "GitTreeState:\"clean\"", version_output
+    # version is reported incorrectly as v4.2.0
+    # see and watch: https://github.com/openshift/oc/issues/416
+
+    # version_output = shell_output("#{bin}/oc version --client 2>&1")
     if build.stable?
-      assert_match "GitVersion:\"v#{version}", version_output
-      assert_match stable.instance_variable_get(:@resource)
-                         .instance_variable_get(:@specs)[:revision].slice(0, 9),
-                   version_output
+      # assert_match "Client Version: v#{version}", version_output
     end
```

Comment:
Can you add some tests for the client?

Reply:
I added the following test:

```ruby
assert_match "Client Version: v4", version_output
```

The expected output of a working `oc` binary is something like:

```
Client Version: v4.2.0-alpha.0-657-g51011e4
```

Due to https://github.com/openshift/oc/issues/416 we currently can't do any better version verification, but at least we can do use this smoke test.